Sources/OpenAI/Internal/Networking/APIRequest.swift

```diff
@@ -16,8 +16,43 @@ enum APIRequestMethod: String {
 
 // MARK: - `APIRequestError` -
 
-enum APIRequestError: Error {
+public enum APIRequestError: Error, LocalizedError {
+    
     case statusCode(Int)
+    case openAIError(OpenAIError)
+    
+    public var localizedDescription: String {
+        switch self {
+        case .openAIError(let msg):
+            print(msg.message)
```

Comment:
This may be a good opportunity to provide custom logging. However, for now, we'll consider it outside the scope of this PR>